generator/generator_tests/osm_type_test.cpp

```diff
@@ -2344,14 +2342,20 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_ComplexTypesSmoke)
     {{"addr:interpolation"}, {{"addr:interpolation", "all"}}},
     {{"aeroway", "aerodrome", "international"}, {{"aeroway", "aerodrome"}, {"aerodrome", "international"}}},
     {{"amenity", "grave_yard", "christian"}, {{"amenity", "grave_yard"}, {"religion", "christian"}}},
+    {{"amenity", "parking"}, {{"amenity", "parking"}, {"fee", "no"}}},
     {{"amenity", "parking", "fee"}, {{"amenity", "parking"}, {"fee", "any_value"}}},
+    {{"amenity", "parking", "lane"}, {{"amenity", "parking"}, {"parking", "lane"}}},
+    {{"amenity", "parking", "lane", "fee"}, {{"amenity", "parking"}, {"parking", "lane"}, {"fee", "any_value"}}},
     {{"amenity", "parking", "multi-storey"}, {{"amenity", "parking"}, {"parking", "multi-storey"}}},
+    {{"amenity", "parking", "multi-storey", "fee"}, {{"amenity", "parking"}, {"parking", "multi-storey"}, {"fee", "any_value"}}},
     {{"amenity", "parking", "no-access"}, {{"amenity", "parking"}, {"access", "no"}}},
```

Comment:
will following work if we add `amenity|parking|multi-storey|private` to the mapping
```
{{"amenity", "parking", "multi-storey", "private"}, {{"amenity", "parking"}, {"parking", "multi-storey"}, {"access", "private"}}},
```
?

Reply:
Yes, same as like with underground in follow-up PR.